cypress/integration/search.spec.js

```diff
@@ -23,3 +23,33 @@ describe('A user searches for a resource', () => {
     cy.url().should('include', '/Search/Norway');
   });
 });
+
+describe('A user searches for a resource and uses pagination', () => {
+  it('Given that a user is on the start page and is not logged in', () => {
+    cy.visit('/');
+    cy.get('[data-cy=menu]').click();
+    cy.get('[data-cy=logout-button]').click();
+    cy.get('[data-cy=login-button]').should('be.visible');
+    cy.get('[data-cy=menu]').should('not.exist');
+  });
+  it('When the user enters Norway into the search input', () => {
+    cy.get('[data-cy=search-input] .MuiInputBase-input').type('Norway');
+  });
+  it('And clicks Search', () => {
+    cy.get('[data-cy=search-button]').click({ force: true });
+  });
+  it('And there is more then 10 results', () => {
+    cy.get('[data-cy=search-results]').contains('25 results');
+  });
+  it('And pagination should show', () => {
+    cy.get('[data-cy=pagination]').contains('2');
```

Comment:
kanskje formulere litt annerledes? dette er vel egentlig en "Then"-statement. "Then the pagination component should be shown"?